File: 3rd Semester/Advanced Programming Methods(Done)/Laboratory/src (BUX)/Model/Utils/Tuple.java
package Model.Utils;

import java.io.BufferedReader;
import java.util.stream.*;

public class Tuple
{
    private String name;
    private BufferedReader bf;

    public Tuple(String name, BufferedReader bf )
    {
        this.bf = bf;
        this.name = name;
    }

    public String getName()
    {
        return this.name;
    }

    public BufferedReader getBf() {
        return this.bf;
    }
}

File: 3rd Semester/Advanced Programming Methods/Laboratory/Toy Interpreter C#/sharplab/Model/Utils/Tuple.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;

namespace sharplab.Model.Utils
{
    class Tuple
    {
        private string name;
        private StreamReader s;

        public Tuple(string name, StreamReader s)
        {
            this.s = s;
            this.name = name;
        }

        public string GetName()
        {
            return this.name;
        }

        public StreamReader GetStream()
        {
            return this.s;
        }
    }
}
